File: module/.android/app/src/main/java/nct/library/flutter/module/host/MainActivity.java
package nct.library.flutter.module.host;

import android.content.Context;

import io.flutter.embedding.android.FlutterActivity;
import io.flutter.embedding.engine.FlutterEngine;

import com.ryanheise.audioservice.AudioServicePlugin;

public class MainActivity extends FlutterActivity {
//    @Override
//    public void configureFlutterEngine(@NonNull FlutterEngine flutterEngine) {
//        GeneratedPluginRegistrant.registerWith(flutterEngine);
//    }
//
    @Override
    public FlutterEngine provideFlutterEngine(Context context) {
        return AudioServicePlugin.getFlutterEngine(context);
    }
}

File: app/src/main/java/com/example/run_flutter_module_in_android_app/MainActivity.kt
package com.example.run_flutter_module_in_android_app

import android.content.Context
import com.ryanheise.audioservice.AudioServicePlugin
import io.flutter.embedding.android.FlutterActivity
import io.flutter.embedding.engine.FlutterEngine


class MainActivity : FlutterActivity() {
    @Nullable
    fun provideFlutterEngine(@NonNull context: Context?): FlutterEngine {
        return AudioServicePlugin.getFlutterEngine(context)
    }
}
